### PR TITLE
Fix paper tab disabled

### PR DESCRIPTION
**Pull request by MarcoUmpierrez**
Hi, I saw that this issue #791 hasn't been fixed yet. Here's a PR to fix it. I've added a test and an example of a disabled tab in the dummy app.

### CODE REVIEW

File: addon/components/paper-tab.js [hunk 1]
```diff
@@ -57,7 +57,9 @@ export default Component.extend(ChildMixin, RippleMixin, FocusableMixin, {
   },
 
   click() {
-    this.sendAction('onClick', ...arguments);
-    this.sendAction('onSelect', this);
+    if(!this.get("disabled")) {
```
Comment (miguelcobain) [inline addon/components/paper-tab.js]: @MarcoUmpierrez could you please change this line to `if (!this.get('disabled')) {` to conform with the rest of the project's coding style?

**Comment by miguelcobain** [comment]
@MarcoUmpierrez this looks awesome. Thank you so much.

There's still something that isn't covered that shouldn't be too hard to do.

A tab can be an anchor if an href is passed. If a tab with an href is disabled, we need to disable a link. But how do we do that?
Well, a simple way is to just don't have any href.
So we can have an intermediate value that "maybe" holds the href if not disabled.

```js
attributeBindings: ['maybeHrefhref'],
maybeHref: computed('href', 'disabled', function() {
  if (this.get('href') && !this.get('disabled')) {
    return this.get('href');
  }
}),
```

Have a test that made sure that no href was rendered for a disabled tab would also be helpful.

**Comment by MarcoUmpierrez** [comment]
@miguelcobain Added the changes you requested and a test to check if the href is set when the tab is disabled.

**Comment by miguelcobain** [comment]
Thanks a lot @MarcoUmpierrez !

**Pull request merged**